app/controllers/pieces_controller.rb

```diff
@@ -27,7 +27,7 @@ def update
         redirect_to game_path(@game)  # redirect to game show page
       end
       format.json do
-        json_result = {valid: true, active: true, not_color: @color}
+        json_result = {valid: true, active: @piece.active, captured: @captured, not_color: @color}
```

Comment:
if you are passing variables into a the hash then they don't need to be instance variables.  You should just be using regular variables.  So your code should look like this.

json_result = {valid: true, active: piece.active, captured: captured, not_color: color}

additionally what you have should not work anyway because you have not defined @captured int the controller.  instance variables declared in the model will not be present in the controller or view.  You must pass them via the the piece object in the form of attr_accessors